v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/KafkaToBigQueryFlex.java

```diff
@@ -206,6 +207,7 @@ public static PipelineResult run(KafkaToBigQueryFlexOptions options)
           ImmutableMap.<String, Object>builder()
               .putAll(kafkaConfig)
               .putAll(setClientAuthConfig(username, password))
+              .putAll(KafkaCommonUtils.configureKafkaOffsetCommit(options))
```

Comment:
I don't think this is the correct place for this. This will only happen if the authentication mode is `SASL_PLAIN`. This should be added outside the if statement.

Reply:
I moved it a different code path. PTAL @Dippatel98